### ISSUE TITLE
accelerate-examples-1.1.0.0 build failure with criterion-1.3.0.0

### ISSUE THREAD

**Issue by DanBurton** [AccelerateHS/accelerate#418]
I am submitting a...
- [x] bug report
- [ ] feature request
- [ ] support request => you might also like to ask your question on the [mailing list](mailto:accelerate-haskell@googlegroups.com) or [gitter chat](https://gitter.im/AccelerateHS/Lobby).

## Description

accelerate-examples-1.1.0.0, when compiled against criterion-1.3.0.0, results in a compile-time error.

## Expected behaviour

Expected to compile.

## Current behaviour

```
[ 2 of 13] Compiling Data.Array.Accelerate.Examples.Internal.Criterion.Config ( lib/Data/Array/Accelerate/Examples/Internal/Criterion/Config.hs, dist/build/Data/Array/Accelerate/Examples/Internal/Criterion/Config.o )

lib/Data/Array/Accelerate/Examples/Internal/Criterion/Config.hs:48:25: error:
    Variable not in scope: forceGC :: Config :-> Bool
   |
48 |             (NoArg (set forceGC False))
   |                         ^^^^^^^
```

## Possible solution (optional)

Update the library to accommodate building against `criterion-1.3` and `criterion-1.4`. Note that `forceGC` was marked as deprecated in `criterion-1.2.*`, and was removed in `criterion-1.3`.

## Steps to reproduce (for bugs)

I was not able to reproduce locally, due to missing some c libs on my local machine. However, you should be able to reproduce the issue by compiling the package against `criterion-1.3.0.0`.

## Your environment

See https://github.com/fpco/stackage/blob/6a8b93b1dfaa70fa3be632c358bcbb5bdaf509a5/debian-bootstrap.sh for details on the docker image used by Stackage to build this package.

**Comment by DanBurton** [comment]
I've removed accelerate-examples from the Stackage nightly build plan, and unfortunately it won't appear in the initial LTS 11 release.

To get back into nightly, open a PR on the Stackage repo as usual. To get into LTS 11, open an issue on https://github.com/fpco/lts-haskell/issues.

**Comment by tmcdonell** [comment]
Thanks for the heads-up, I had not noticed the deprecation warnings. I'll upload a new version shortly.

**Comment by tmcdonell** [comment]
fixed with 1.2 release
